### PR TITLE
Fix - Federal Account Table Columns

### PR DESCRIPTION
**Pull request by mab6bf**
Does not need to go in this deploy.

- Columns in the Federal Account table now match those in the Award Search table for each Award Type

Bug: https://federal-spending-transparency.atlassian.net/browse/DEV-116

Note: For full testing, @michaeldhess is looking into finding a Federal Account in the dev database that has corresponding awards to display in the table. However, even without result data in the table (which allows the user to scroll through the column headers), the visible column headers can be seen updating on each tab change.

Use 2421

### CODE REVIEW

**Comment by kevinli-work** [comment]
Pending getting a federal account to test this with.

**Comment by kevinli-work** [comment]
* Award count tabs are all 0 (if we don't have the counts, can you hide the count badge?)
* Awarding Agency/Subagency doesn't appear:
screen shot 2018-01-19 at 12 58 14 pm

* Loans tab causes a 500 error: `Cannot resolve keyword 'face_value_loan_guarantee' into field`

**Comment by kevinli-work** [comment]
* Sorting by any field other than award amount returns 0 results

File: src/js/components/account/awards/LegacyResultsTable.jsx [hunk 1]
```diff
@@ -8,16 +8,15 @@
...
-import { formatMoney } from 'helpers/moneyFormatter';
-
 import IBTable from 'components/sharedComponents/IBTable/IBTable';
 
 import ResultsTableGenericCell from 'components/search/table/cells/ResultsTableGenericCell';
 import ResultsTableAwardIdCell from 'components/search/table/cells/ResultsTableAwardIdCell';
 
 import LegacyTableHeaderCell from './LegacyTableHeaderCell';
 
+import AccountTableSearchFields from '../../../dataMapping/search/accountTableSearchFields';
```
Comment (kevinli-work) [inline src/js/components/account/awards/LegacyResultsTable.jsx]: o rly? You should be able to use the module path from `src/js/`

File: src/js/containers/account/awards/AccountAwardsContainer.jsx [hunk 2]
```diff
@@ -11,15 +11,16 @@ import { uniqueId } from 'lodash';
...
+import AccountTableSearchFields from 'dataMapping/search/accountTableSearchFields';
 import { awardTypeGroups } from 'dataMapping/search/awardType';
 import { awardTableColumnTypes } from 'dataMapping/search/awardTableColumnTypes';
 import * as SearchHelper from 'helpers/searchHelper';
 
 import AccountAwardSearchOperation from 'models/account/queries/AccountAwardSearchOperation';
-
 import AccountAwardsSection from 'components/account/awards/AccountAwardsSection';
 
+import BaseFederalAccountAwardRow from '../../../models/v2/BaseFederalAccountAwardRow';
```
Comment (kevinli-work) [inline src/js/containers/account/awards/AccountAwardsContainer.jsx]: See above comment about module paths

File: src/js/components/account/awards/LegacyResultsTable.jsx (hunk 1 above)
Comment (mab6bf) [reply src/js/components/account/awards/LegacyResultsTable.jsx]: Fixed

File: src/js/containers/account/awards/AccountAwardsContainer.jsx (hunk 2 above)
Comment (mab6bf) [reply src/js/containers/account/awards/AccountAwardsContainer.jsx]: Fixed

**Pull request merged**